src/modules/sensors/sensors.cpp

```diff
@@ -273,6 +273,11 @@ class Sensors
 	uint64_t _last_mag_timestamp[SENSOR_COUNT_MAX]; /**< latest full timestamp */
 	uint64_t _last_baro_timestamp[SENSOR_COUNT_MAX]; /**< latest full timestamp */
 
+	float _accel_diff[3][2];	/**< filtered accel differences between IMU units (m/s/s) */
```

Comment:
I've limited it to checking against no more than two other sensors here: 

https://github.com/priseborough/Firmware/blob/sensorDataValidation-wip/src/modules/sensors/sensors.cpp#L2290 

and here:

https://github.com/priseborough/Firmware/blob/sensorDataValidation-wip/src/modules/sensors/sensors.cpp#L2341